docs/e-comerce-erd.sql

```diff
@@ -0,0 +1,85 @@
+
+CREATE TABLE tb_balance
+(
+  user_id BIGINT   NOT NULL COMMENT '사용자고유ID',
+  balance BIGINT   NOT NULL DEFAULT 0 COMMENT '사용자별 보유 금액',
+  reg_dt  DATETIME NOT NULL DEFAULT now() COMMENT '등록일',
+  upd_dt  DATETIME NULL     COMMENT '수정일'
+) COMMENT '보유금액';
+
+CREATE TABLE tb_balance_history
+(
+  id           BIGINT   NOT NULL DEFAULT auto_increment COMMENT 'PK',
+  user_id      BIGINT   NOT NULL COMMENT '사용자고유ID',
+  amount       BIGINT   NOT NULL COMMENT '충전 또는 사용 금액',
+  balance_type ENUM     NOT NULL COMMENT '1: 충전, 2: 사용',
+  reg_dt       DATETIME NOT NULL DEFAULT now() COMMENT '등록일',
+  PRIMARY KEY (id)
+) COMMENT '사용자 금액 내역';
+
+CREATE TABLE tb_coupon
+(
+  id            BIGINT   NOT NULL DEFAULT auto_increment COMMENT '쿠폰고유ID',
+  discount_rate INT      NOT NULL DEFAULT 0 COMMENT '할인율',
+  status        INT      NOT NULL DEFAULT 0 COMMENT '발급여부 1: 발급됨, 0: 발급전',
+  reg_dt        DATETIME NOT NULL DEFAULT now() COMMENT '등록일',
+  upd_dt        DATETIME NULL     COMMENT '수정일',
+  PRIMARY KEY (id)
+) COMMENT '선착순쿠폰';
+
+CREATE TABLE tb_order
+(
+  id               BIGINT   NOT NULL DEFAULT auto_increment COMMENT '주문고유ID',
+  user_id          BIGINT   NOT NULL COMMENT '사용자고유ID',
+  coupon_id        BIGINT   NULL     COMMENT '쿠폰고유ID',
+  order_price      BIGINT   NOT NULL DEFAULT 0 COMMENT '주문가격',
+  last_order_price BIGINT   NOT NULL COMMENT '최종주문가격(쿠폰적용)',
+  status           INT      NOT NULL DEFAULT 0 COMMENT '0: 대기, 1:성공, 2:실패',
```

Comment:
```suggestion
  order_price      BIGINT   NOT NULL DEFAULT 0 COMMENT '주문가격',
  final_order_price BIGINT   NOT NULL COMMENT '최종주문가격(쿠폰적용)',
  status           INT      NOT NULL DEFAULT 0 COMMENT '0: 대기, 1:성공, 2:실패',
```
최종주문가격의 의미로 `final_order_price`가 더 좋아 보입니다!

Reply:
감사합니다!